lib/clearance/password_strategies/sha1.rb

```diff
@@ -16,7 +16,7 @@ module SHA1
       #   gem
       def authenticated?(password)
         warn "#{Kernel.caller.first}: #{DEPRECATION_MESSAGE}"
-        encrypted_password == encrypt(password)
+        encrypted_password == _encrypt(password)
```

Comment:
The blowfish password strategy is deprecated. Please do not make any changes to this file. Thanks!